client/src/components/Home.js

```diff
@@ -62,7 +62,7 @@ import {
 //   </svg>
 // )
 
-const style = (theme) => ({
+const style = () => ({
```

Comment:
So changes like this are for the purpose that we can pass the lint checking? 
I am thinking that maybe the problem like violate the rule of code style and lint warning, is not necessary to check for the code PR commit and deploy, what do you think?

Reply:
Although some lint warnings are related to code style, this step is mostly there to catch potential bugs with static analysis. For example, an unused parameter like this might indicate a scoping problem within the function, or where the developer has forgotten to read the input and has used a member variable instead.

The workflow doesn't run `prettier`, just `eslint`. Warnings and errors from `eslint` should be addressed before the code is deployed, even if the appropriate _solution_ is to suppress a warning.